### ISSUE TITLE
incorrect grouping for permission check

### ISSUE THREAD

**Issue by digitalirony** [AndrykVP/discord-raffle#1]
https://github.com/AndrykVP/discord-raffle/blob/faa1c54f3ee5db5b78c35e328ec702fc70c06e6b/commands/start.js#L16

Should be like this:
 if(!message.member.roles.some(r => admins.includes(r.name)) || !admins.includes(message.author.username))

(you are missing the 2nd ) after r.name. This causes it to pretty much always return false.

**Comment by aj1444** [comment]
{
    "prefix": "!",
    "admins": [
        "admin"
    ],
    "color": "8A1E22"
} 
here even if he put it its not starting the raffle

**Comment by digitalirony** [comment]
@aj1444 what is the error you get in the console?

**Comment by aj1444** [comment]
> @aj1444 what is the error you get in the console?

 npm start

discord-raffle@1.0.0 start /home/runner/discord-raffle-1
node .

Bot is running
TypeError: Cannot read property 'roles' of null
    at Object.execute (/home/runner/discord-raffle-1/commands/start.js:16:29)
    at Client.<anonymous> (/home/runner/discord-raffle-1/index.js:37:36)
    at Client.emit (events.js:314:20)
    at MessageCreateHandler.handle (/home/runner/discord-raffle-1/node_modules/discord.js/src/client/websocket/packets/handlers/MessageCreate.js:9:34)
    at WebSocketPacketManager.handle (/home/runner/discord-raffle-1/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:105:65)
    at WebSocketConnection.onPacket (/home/runner/discord-raffle-1/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:333:35)
    at WebSocketConnection.onMessage (/home/runner/discord-raffle-1/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:296:17)
    at WebSocket.onMessage (/home/runner/discord-raffle-1/node_modules/ws/lib/event-target.js:120:16)
    at WebSocket.emit (events.js:314:20)
    at Receiver.receiverOnMessage (/home/runner/discord-raffle-1/node_modules/ws/lib/websocket.js:789:20)

**Comment by aj1444** [comment]
@digitalirony  TypeError: Cannot read property 'roles' of null
    at Object.execute (/home/runner/mrraffle/commands/start.js:18:28)
    at Client.<anonymous> (/home/runner/mrraffle/index.js:46:36)
    at Client.emit (events.js:314:20)
    at MessageCreateHandler.handle (/home/runner/mrraffle/node_modules/discord.js/src/client/websocket/packets/handlers/MessageCreate.js:9:34)
    at WebSocketPacketManager.handle (/home/runner/mrraffle/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:105:65)
    at WebSocketConnection.onPacket (/home/runner/mrraffle/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:333:35)
    at WebSocketConnection.onMessage (/home/runner/mrraffle/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:296:17)
    at WebSocket.onMessage (/home/runner/mrraffle/node_modules/ws/lib/event-target.js:120:16)
    at WebSocket.emit (events.js:314:20)
    at Receiver.receiverOnMessage (/home/runner/mrraffle/node_modules/ws/lib/websocket.js:789:20)
    at Receiver.emit (events.js:314:20)
    at Receiver.dataMessage (/home/runner/mrraffle/node_modules/ws/lib/receiver.js:422:14)
    at Receiver.getData (/home/runner/mrraffle/node_modules/ws/lib/receiver.js:352:17)
    at Receiver.startLoop (/home/runner/mrraffle/node_modules/ws/lib/receiver.js:138:22)
    at Receiver._write (/home/runner/mrraffle/node_modules/ws/lib/receiver.js:74:10)
    at doWrite (_stream_writable.js:403:12)
    
    
    this error coming please help me

**Comment by digitalirony** [comment]
@aj1444 Can you try giving it at least one specific username, and then the roles?

**Comment by aj1444** [comment]
@digitalirony {
    "prefix": "!",
    "admins": [
        "Eren-Yeager",
	"GM"
    ],
    "color": "8A1E22"
}
 
 
 this is the command in config.json

**Comment by aj1444** [comment]
@digitalirony https://discord.gg/jgaUwEkfU7 please join this server and help me

**Comment by digitalirony** [comment]
We discussed this on discord, and after some testing this was determined to be an issue between replit and how the code is used on a 'bare metal' system.
The issue was with how replit does module packages with npm.
I was able to fix this by adding the dependency list directly to package.json(instead of the package-lock.json)
and making sure that replit had deleted all of the packages that it installed at later versions.
